modules/@apostrophecms/notification/index.js

```diff
@@ -209,17 +209,17 @@ module.exports = {
           action: self.action
         };
       },
-      // Call with `req`, then a message key as found in the localization files,
-      // followed by an `options` object if desired.
-      //
-      // If you do not have a `req` it is acceptable to pass a user `_id` string
+      // When used server-side, call with `req` as the first argument,
+      // or if you do not have a `req` it is acceptable to pass a user `_id` string
       // in place of `req`. Someone must be the recipient.
       //
-      // `message` should be a key that exists in a localization file. If it does not
-      // it will be displayed directly as a fallback.
+      // When called client-side, this first argument should be ommitted.
```

Comment:
Should be more strongly worded, and there is a typo.

"When called client-side, there is no `req` argument because the recipient is always the current user."